crates/apollo-parser/src/parser/mod.rs

```diff
@@ -103,9 +103,9 @@ pub struct Parser<'a> {
 /// Defaulting to around a quarter of that, to keep a comfortable safety margin.
 const DEFAULT_RECURSION_LIMIT: usize = 500;
 
-impl<'a> Parser<'a> {
+impl<'input> Parser<'input> {
```

Comment:
Renamed the lifetime so it's clear what it refers to when you're looking at a return value somewhere a few pages down in the impl block